### DISCUSSION TITLE
Selecting A Global Font

### DISCUSSION THREAD

**Discussion by jubalm** [DarkFlorist/TheInterceptor#973] [General]
## Introduction

A discussion for The Interceptor getting a default font for the interface is in the works which aims to improve the browser extension's overall user experience, by improving text legibility as well as consistency across different browsers and operating systems.

## Font Candidates

Selection was based on a few basic traits of a font.

- Web-friendly
- Distinction with default system fonts
- Legibility / Kerning
- File size

#### Atkinson Hyperlegible - https://brailleinstitute.org/freefont

<img width="504" alt="Screenshot 2024-04-16 at 2 16 55 PM" src="https://github.com/DarkFlorist/TheInterceptor/assets/1169838/8f3f48b9-cf61-42f8-be2d-ad7f15d538f8">

- created by the Braille foundation with strong focus on readability
- numbers are pronounced with zero having a "slash" in the middle
- file size is ~68kb for three font weights (woff2)

#### Inter (Google Font) - https://fonts.google.com/specimen/Inter

<img width="511" alt="Screenshot 2024-04-16 at 2 17 22 PM" src="https://github.com/DarkFlorist/TheInterceptor/assets/1169838/eba06ef4-ad59-4821-bb5f-534c21ce10e8">

- widely used
- tight letter-spacing, good for narrow interfaces
- ~800kb for the variable font (direct download)

#### Satoshi - https://www.fontshare.com/fonts/satoshi

<img width="511" alt="Screenshot 2024-04-16 at 2 18 37 PM" src="https://github.com/DarkFlorist/TheInterceptor/assets/1169838/768ad021-a879-4865-89ee-02ce52133eb0">

- elegant thin letters but very legible
- variable ~80kb, including italic

**Comment by MicahZoltu** [comment]
The Google one seems unreasonably large (file size).  Why is this?

**Comment by KillariDev** [comment]
For me its hard to judge from these pictures. Could we have a pr with the font and we could see what it looks like?
